src/translator/text-based-field.ts

```diff
@@ -166,10 +166,10 @@ export namespace TextBasedField {
     ) => {
         const { showIf, uiType, validation, label, ...rest } = element;
 
-        let requiredValidation: IYupValidationRule = {};
+        let requiredValidation: IYupValidationRule = { required: false };
```

Comment:
this isnt necessary, you can cast the value as boolean on line 186

```
required: !!requiredValidation.required,
```

with this, even if `requiredValidation` is assigned an invalid value subsequently, this will still convert it in true / false